### PR TITLE
install.md: openSUSE dependencies

### PR DESCRIPTION
**Pull request by jvanz**
Adds documentation about how to install dependencies on openSUSE

Signed-off-by: José Guilherme Vanz <jvanz@jvanz.com>

### CODE REVIEW

**Comment by giuseppe** [comment]
LGTM

@saschagrunert FYI

File: install.md [hunk 1]
```diff
@@ -149,6 +149,12 @@ sudo apt-get install \
   uidmap
 ```
 
+On openSUSE Leap:
```
Comment (saschagrunert) [inline install.md]: Is this bound to any version, like Leap > 15? Do we want to add a note about Tumbleweed as well? :)

**Comment by rhatdan** [comment]
/approve
LGTM
@jvanz Can you fix @saschagrunert comments?

File: install.md (hunk 1 above)
Comment (jvanz) [reply install.md]: Done. :)

**Comment by jvanz** [comment]
> /approve
> LGTM
> @jvanz Can you fix @saschagrunert comments?

Done. :)

**Pull request merged**